docs/playbook-reference/actions/scans.rst

```diff
@@ -0,0 +1,48 @@
+Scans
+########
+
+Robusta includes built-in actions to scan and get insights on Kubernetes clusters.
+
+These actions can be triggered automatically on a schedule, manually using the Robusta Platform and :ref:`manually using the robusta cli <Manual Triggers>`.
+
+
+Popeye - A Kubernetes Cluster Sanitizer
+---------------------------------------------
+
+`Popeye <https://github.com/derailed/popeye>`_ is a utility that scans live Kubernetes cluster and reports potential issues with deployed resources and configurations. by default Robusta will run a Popeye scan on startup.
```

Comment:
Suggested change below. My changes are in bold (but don't keep the bold - just using it to highlight what I changed.)

`Popeye <https://github.com/derailed/popeye>`_ is a utility that scans Kubernetes **clusters** and reports **potential issues with resources and configurations**.

**By default, every instance of Robusta that's connected to the UI will run a Popeye scan on startup. Further Popeye scans can be triggered in the UI, and all scans can be viewed there.**